includes/sanitizers/class-amp-style-sanitizer.php

```diff
@@ -2661,23 +2703,41 @@ private function process_font_face_at_rule( AtRuleSet $ruleset, $options ) {
 					'genericons.woff',
 				];
 				if ( in_array( $font_filename, $bundled_fonts, true ) ) {
-					$data_url->getURL()->setString( plugin_dir_url( AMP__FILE__ ) . "assets/fonts/$font_filename" );
-					$converted_count++;
+					$font_file = plugin_dir_url( AMP__FILE__ ) . "assets/fonts/$font_filename";
+					$data_url->getURL()->setString( $font_file );
+					if ( 'inline' === $first_src_type ) {
+						$first_src_type = 'file';
+						$font_file      = $font_file;
+					}
```

Comment:
Humm, `$font_file = $font_file`?

I think this can be just:

```suggestion
					$font_file = plugin_dir_url( AMP__FILE__ ) . "assets/fonts/$font_filename";
					$data_url->getURL()->setString( $font_file );
					$first_src_type = 'file';
```

Reply:
Done in 6c1611367ad12c831df0f0492c25c06544575dd7